cypress/e2e/web/technologyDetails.js

```diff
@@ -101,4 +101,73 @@ describe('technology details', () => {
 		cy.url().should('match', /_error.js/);
 		cy.findAllByText('404').should('exist');
 	});
+
+	describe('add review', () => {
+		beforeEach(() => {
+			cy.visit(`/t/${technology.slug}`);
+			cy.signIn();
+
+			cy.findAllByText(/relatos de experiência/i)
+				.should('exist')
+				.click();
+
+			cy.findAllByRole('button', { name: /adicionar tecnologia/i })
+				.should('exist')
+				.click();
+		});
+
+		it('should close modal when user does not have the technology', () => {
+			cy.findAllByText(/ainda não/i)
+				.should('be.visible')
+				.click();
+
+			cy.findAllByText(/entendi/i)
+				.should('be.visible')
+				.click();
+
+			cy.findAllByText(/você já tem essa tecnologia/i).should('not.exist');
+			cy.findAllByText(/espere um pouco/i).should('not.exist');
+		});
+
+		it('should add a new technology review', () => {
+			const data = {
+				content: 'any content',
+				positive: 'any text',
+				negative: 'any text',
+			};
+			const modalTitle = /Como foi sua experiência com essa tecnologia?/;
+
+			cy.findAllByText(/sim, já tenho/i)
+				.should('be.visible')
+				.click();
+
+			cy.findAllByText(modalTitle).should('be.visible');
+
+			// Add points
```

Comment:
Só uma frescura aqui mas `points` não faz tanto sentido em inglês aqui. Eu trocaria por item ou só deixaria "add positives and negatives".

Reply:
Tranquilo, vou alterar :slightly_smiling_face: